fab/fabfile.py

```diff
@@ -489,38 +494,35 @@ def development():
     env.django_port = '9010'
     env.should_migrate = True
 
-    require('inventory')
-
-    # use inventory filename as environment name
-    # i.e. if the inventory is called my-crazy-setup
-    # then things on the server will be stored in
-    # /home/cchq/www/my-crazy-setup/code_root, etc.
-    env.environment = os.path.basename(env.inventory)
+    require('inventory', 'environment')
     servers = read_inventory_file(env.inventory)
 
     _setup_path()
 
+    proxy = servers['proxy']
     webworkers = servers['webworkers']
     postgresql = servers['postgresql']
     couchdb = servers['couchdb']
     redis = servers['redis']
     memcached = servers['memcached']
+    touchforms = servers['touchforms']
+    elasticsearch = servers['elasticsearch']
```

Comment:
Yup, I wanted to have a variable in here set for every server type we have in ansible rather than having to figure that out later